Src/DependencyCollector/Microsoft.AI.DependencyCollector.Tests/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "1.0.0-*",
+
+  "testRunner": "mstest",
+
+  "buildOptions": {
+    "keyFile": "../../../Keys/35MSSharedLib1024.snk",
+    "delaySign": true,
+    "xmlDoc": true
+  },
+
+  "dependencies": {
+    "dotnet-test-mstest": "1.1.2-preview",
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.1"
+    },
+    "MSTest.TestFramework": "1.0.8-rc2",
+    "Microsoft.AI.DependencyCollector": {
+      "target": "project",
+      "version": "2.3.0-beta2"
+    }
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": "dnxcore50"
```

Comment:
This was the default that the tool created when I created a new ASP.NET project. I tried to remove it, but that created a build failure. It looks like that allows me to run .NET Core apps on the .NET Core framework.